modin/pandas/base.py

```diff
@@ -3127,6 +3127,56 @@ def __init__(
         ]
         self.__groups = self.__get_groups(*self.resample_args)
 
+    _attributes = [
+        "rule",
+        "axis",
+        "closed",
+        "label",
+        "convention",
+        "kind",
+        "loffset",
+        "base",
+        "on",
+        "level",
+        "origin",
+        "offset",
+    ]
+
+    def __getitem__(self, key):
+        if len(self._dataframe.columns.intersection(key)) != len(key):
+            bad_keys = list(set(key).difference(self._dataframe.columns))
+            raise KeyError(f"Columns not found: {str(bad_keys)[1:-1]}")
+        return self._gotitem(list(key), ndim=2)
+
+    def _gotitem(self, key, ndim: int):
+        """
+        Get sub-classes to define.
+
+        Parameters
+        ----------
+        key : string or list
+            String or list of selections.
+        ndim : {1, 2}
+            Requested ndim of result.
+
+        Returns
+        -------
+        modin.pandas.BasePandasDataset
+            Sliced object.
+        """
+        kwargs = {
+            self._attributes[i]: self.resample_args[i]
+            for i in range(len(self._attributes))
+        }
```

Comment:
Can be resolved as part of `self.resample_args` -> dict fixing in __init__ method.